### DISCUSSION TITLE
Info about Skylicht Engine

### DISCUSSION THREAD

**Discussion by bendico1** [skylicht-lab/skylicht-engine#114] [General]
Hi,
first of all, congrats for your great work about Skylicht Engine!
I'm a beginner in C++ and in game development, but I already used a little bit of Irrlicht for learning purpose. Recently, I discovered Skylicht and I like it very much!!! Its graphics is impressive and it looks very interesting and promising from Irrlicht's user point of view!
I set up my own Visual Studio project and I began to experiment with Skylicht.
So my question is: what about the Physics?
Is there a Physics implementation system among other Skylicht features?
Thanks for your kindly support.
Bye!

**Comment by ducphamhong** [comment]
Hello, Sorry because late response.
Skylicht still in development.
About physics engine: I will integrate the BulletPhysics3 soon. But i am focusing on the 3D World Editor first.
Thanks
Pham Hong Duc

**Comment by RobotGamez** [comment]
I could not compile the code with Visual Studio 2022.
It couldnt find some key thing....

Can you explain clearer how to run this in Cmake and then in Visual Studio - best to do a video so we can follow...............
What do you mean when you say build assets;   C:\skylicht-engine>cd Assets

The number of errors I got was about 360 until I went into folder 'Assets' and ran 'Build.cmd', and now I only get like 2 errors
- one about cant find function VLDEnable & VLDDisable (CBaseApp.cpp) , plus cant open SDL2.lib file (TestApp project).

Can you clarify how to fix this please ?????

Even when I 'Set as startup project' -  it says 'cannot set the pixel format' error message.........
